build.ps1

```diff
@@ -35,7 +41,13 @@ if($Mono.IsPresent) {
 
 # Try download NuGet.exe if do not exist.
 if (!(Test-Path $NUGET_EXE)) {
-    Invoke-WebRequest -Uri http://nuget.org/nuget.exe -OutFile $NUGET_EXE
+    try{
+        Invoke-WebRequest -Uri http://nuget.org/nuget.exe -OutFile $NUGET_EXE
```

Comment:
Put nuget URL in $NUGET_URL variable so same it's used for both Invoke-WebRequest & WebClient, I prefer https URL is used.